java_console/io/src/main/java/com/rusefi/ldmp/StateDictionary.java

```diff
@@ -27,6 +27,7 @@
         register(live_data_e.LDS_AC_CONTROL, AcControl.VALUES); // 12
         register(live_data_e.LDS_FUEL_PUMP, FuelPump.VALUES);
         register(live_data_e.LDS_TPS_ACCEL, TpsAccelState.VALUES);
+		register(live_data_e.LDS_MAIN_RELAY, MainRelay.VALUES);
```

Comment:
code formatting

I will now fix this in master in a proper fancy way with a unit test